wiktextract/form_descriptions.py

```diff
@@ -2487,6 +2496,48 @@ def classify_desc(desc, allow_unknown_tags=False, no_unknown_starts=False):
               cnt >= len(lst) - 1) or
              cnt / len(lst) >= 0.8)):
             return "english"
+        lst = list(x not in not_french_words and
+                   # not x.isdigit() and
+                   (x in french_words or x.lower() in french_words or
+                    x in known_firsts or
+                    x[0].isdigit() or
+                    # (x[0].isupper() and x.find("-") < 0 and x.isascii()) or
+                   (x.endswith("s") and len(x) >= 4 and
+                    x[:-1] in french_words) or  # Plural
+                    (x.endswith("ies") and len(x) >= 5 and
+                     x[:-3] + "y" in french_words) or  # E.g. lily - lilies
+                    (x.endswith("ing") and len(x) >= 5 and
+                     x[:-3] in french_words) or  # E.g. bring - bringing
+                    (x.endswith("ing") and len(x) >= 5 and
+                     x[:-3] + "e" in french_words) or  # E.g., tone - toning
+                    (x.endswith("ed") and len(x) >= 5 and
+                     x[:-2] in french_words) or   # E.g. hang - hanged
+                    (x.endswith("ed") and len(x) >= 5 and
+                     x[:-2] + "e" in french_words) or  # E.g. atone - atoned
+                    (x.endswith("'s") and x[:-2] in french_words) or
+                    (x.endswith("s'") and x[:-2] in french_words) or
+                    (x.endswith("ise") and len(x) >= 5 and
+                     x[:-3] + "ize" in french_words) or
+                    (x.endswith("ised") and len(x) >= 6 and
+                     x[:-4] + "ized" in french_words) or
+                    (x.endswith("ising") and len(x) >= 7 and
+                     x[:-5] + "izing" in french_words) or
+                    (re.search(r"[-/]", x) and
+                     all(((y in french_words and len(y) > 2)
+                          or not y)
+                         for y in re.split(r"[-/]", x))))
+                   for x in tokens)
```

Comment:
Did you just copy and paste this long block of code from the above lines? This code and the 8M `french_words.txt` file and `french_words.py` file seems unnecessary.